internal/pkg/describe/env.go

```diff
@@ -120,6 +121,41 @@ func (d *EnvDescriber) Version() (string, error) {
 	return metadata.Version, nil
 }
 
+// EnvironmentVPC holds the ID of the environment's VPC configuration.
+type EnvironmentVPC struct {
+	ID               string
+	PublicSubnetIDs  []string
+	PrivateSubnetIDs []string
+}
+
+// EnvironmentVPC returns the ID of the VPC and its subnets by reading the outputs of the environment cloudformation stack.
+func (d *EnvDescriber) EnvironmentVPC() (*EnvironmentVPC, error) {
```

Comment:
Do we use this function?

Reply:
🤦 same as above, it was code that I had written while experimenting with the cmd. Deleted it :P